libs/signers/src/fromDisk.ts

```diff
@@ -18,7 +18,7 @@ function loadEncrypted(name: AccountName) {
     if (path) {
       if (!existsSync(path))
         throw new Error(`Could not load ${name} from path: ${path}`);
-      wallet = readFileSync(path, 'utf8');
+      wallet = readFileSync(path, 'ascii');
```

Comment:
ascii instead of utf8?

Reply:
yeah, one of my accounts was loading with a weird leading character in it that broke JSON parsing.  I have no idea how it got in there, I assume that at some point the file was saved as Unicode and that was the leading byte for it.  Forcing it to `ASCII` strips the byte.   This kinda runs counter to most of what I thought I knew about unicode, but...

These are machine-files, it doesn't include any user-generated content, so the Utf8 is not necessary.